pkgs/os-specific/linux/syslinux/default.nix

```diff
@@ -16,11 +18,10 @@ stdenv.mkDerivation {
 
   # This is syslinux-6.04-pre3^1; syslinux-6.04-pre3 fails to run.
   # Same issue here https://www.syslinux.org/archives/2019-February/026330.html
-  src = fetchgit {
-    url = "https://repo.or.cz/syslinux";
-    rev = "b40487005223a78c3bb4c300ef6c436b3f6ec1f7";
-    sha256 = "sha256-GqvRTr9mA2yRD0G0CF11x1X0jCgqV4Mh+tvE0/0yjqk=";
-    fetchSubmodules = true;
+  src = fetchFromRepoOrCz {
```

Comment:
Didn't know that existed. fetchgit defaults to fetching submodules, I assume this doesnt

Reply:
It also uses a zip instead of git.